docs/reference/cli/index.md

```diff
@@ -3422,6 +3422,63 @@ You can overwrite the file while Teku is running to update the graffiti.
 
 This option takes precedence over [`--validators-graffiti`](#validators-graffiti).
 
+### `validators-graffiti-client-append-format`
+
+<Tabs>
+  <TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--validators-graffiti-client-append-format=<STRING>
+```
+
+  </TabItem>
+  <TabItem value="Example" label="Example" >
+
+```bash
+--validators-graffiti-client-append-format=CLIENT_CODES
+```
+
+  </TabItem>
+  <TabItem value="Environment variable" label="Environment variable" >
+
+```bash
+TEKU_VALIDATORS_GRAFFITI_CLIENT_APPEND_FORMAT=CLIENT_CODES
+```
+
+  </TabItem>
+  <TabItem value="Configuration file" label="Configuration file" >
+
+```bash
+validators-graffiti-client-append-format: CLIENT_CODES
+```
+
+  </TabItem>
+</Tabs>
+
+Appends CL and EL clients information with a space to user's graffiti. On
+a separate BN/VC setup should be set on a Beacon Node. This feature helps
```

Comment:
```suggestion
Configure the beacon node and validator client setup on a separate Beacon Node. This feature helps
```

Reply:
No, it means if you run it separately, you should add a flag or change it only on BN. It will not affect VC settings. If you run single node, you do it on this single node, no questions.